src/main/java/edu/harvard/iq/dataverse/datavariable/VariableCategory.java

```diff
@@ -153,21 +153,6 @@ public void setFrequency(Double frequency) {
     // helper for html display  
     // [TODO: double-check if we still need this method in 4.0; -- L.A., jan. 2014] 
     private transient List charList;
-
-    public List getValueCharacterList() {
-        if (charList == null) {
-            charList = new ArrayList();
-            for (int i=0; i < this.value.length(); i++) {
-                if (this.value.charAt(i) == ' ') {
-                    charList.add( "&nbsp;" );
-                } else {
-                    charList.add( this.value.charAt(i) );
-                }
-            }
-        }
-        return charList;
-    }
-    
```

Comment:
Removed as it was suggested we didn't need this, it isn't used anywhere, and the author thought it would be safe to remove.